depletion/casl_chain.py

```diff
@@ -278,3 +275,9 @@
     'Dy165': (False, 3, 2, None),
     'Ho165': (True, 3, 3, [('Dy165_m1', 0.022, 2), ('Ho165', 1.000, 1)])
 }
+
+# When building the depletion chain, we generally ensure that the sum of
+# branching ratios is one. However, in a few cases, this doesn't make sense and
+# would end up producing incorrect concentrations downstream by overproducing
+# certain nuclides.
```

Comment:
Looking at what is contained in the CASL chain and the various isotope decay modes (through wikipedia), the products for some of the decay modes don't exist in the chain. We shouldn't scale the Tm170 -> Er170 branching ratio from 0.14% to one because Yb doesn't exist, definitely. But, if someone needs to alter this code in the future, the comment should (IMO) describe why the isotopes are included and the criteria for adding or rejecting other isotopes

Reply:
Good point; it is rather subtle. I'll beef up the description here

Reply:
Ok, just added an explanation of one of the cases